package.json

```diff
@@ -4,7 +4,7 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "test": "jest",
+    "test": "./node_modules/.bin/jest",
```

Comment:
As `npm install` doesn't install `jest` globally, this will reference the copy we have in the `node_modules` directory and work both locally and on CI. Anything we can do to make our local setup run in the same way as our CI setup makes the whole process more reliable. Noting is more frustration than CI failing whilst all our tests pass locally!